src/components/PersonLink/PersonLink.tsx

```diff
@@ -0,0 +1,52 @@
+import { Link, useParams } from 'react-router-dom';
+import { Person } from '../../types';
+import classNames from 'classnames';
+
+type Props = {
+  person: Person;
+};
+export const PersonLink: React.FC<Props> = ({ person }) => {
+  const { slug } = useParams();
+  const { name, sex, born, died, fatherName, motherName, mother, father } =
+    person;
+  const female = sex === 'f';
+  const notParents = '-';
+
+  return (
+    <tr
+      data-cy="person"
+      className={classNames({ 'has-background-warning': slug === person.slug })}
+    >
+      <td>
+        <Link
+          to={person.slug}
+          className={classNames({
+            'has-text-danger': female,
+          })}
+        >
+          {name}
+        </Link>
```

Comment:
Create `PersonLink` component for this, and use it here and in places below where you have links